config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/jaegertracing/jaeger-operator
-  newTag: v1.28.0
+  newName: quay.io/rvargasp/jaeger-operator
+  newTag: v1.28.0-14-gbcb7b681
```

Comment:
Any reason not to use the official image here?

Reply:
No, this is a mistake